pages/main_page.py

```diff
@@ -0,0 +1,77 @@
+import time
+from selenium.webdriver import Keys
+from .locators import *
+from bdd_task.support.browser_helper import FindElement, ElementStatements
+from bdd_task.support.date_picker_helper import DatePickerLocators
+
+
+class MainPage(FindElement, ElementStatements):
+    def accept_cookies(self):
+        accept_cookies_button = self.find_clickable_element(*MainPageLocators.ACCEPT_COOKIES_BUTTON)
+        accept_cookies_button.click()
+
+    def enter_departure_location(self, departure_location):
+        departure_location_field = self.find_clickable_element(*MainPageLocators.DEPARTURE_LOCATION_SEARCH_FIELD)
+        departure_location_field.click()
```

Comment:
here only click is enough, everything else should be under the hood (like waits and so on)